tests/sentry/api/serializers/test_release.py

```diff
@@ -410,6 +412,52 @@ def test_release_no_users(self):
         release.add_project(project)
         serialize(release)
 
+    def test_get_user_for_authors_simple(self):
+        user = User.objects.create(email="chrib@sentry.io")
+        project = self.create_project()
+        self.create_member(user=user, organization=project.organization)
+        users = get_users_for_authors(organization_id=project.organization_id, authors=[user])
```

Comment:
This function accepts a list of `CommitAuthor` as authors, not `User`. It happens to work because both have an `email` attribute, but this isn't testing the intended functionality